package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "star-rating",
   "widgetName": "StarRating",
-  "version": "1.1.3",
+  "version": "1.1.4",
```

Comment:
The is a new feature, so it should not be a patch update.
=> `1.2.0`

Reply:
Was done